### ISSUE TITLE
custom  object  collision issue

### ISSUE THREAD

**Issue by casigus** [TheGameCreators/GameGuruRepo#2842]
collision in the editor works.

![image](https://user-images.githubusercontent.com/68471279/163260840-07329293-e2e4-4bc9-89c0-18d7d1ddbfb5.png)

collision in the  game  doesn't.

NOW

whats also intresting is that the models that the textures  have justt disapeared witth  (even though the textures are in folder) do have collision.  

now after trying too import the model again (and having  to do the texture work manualllyl) tthe  model   suddenly has no collision.

I'm utterly stumped. it worked perfectly last night.

![image](https://user-images.githubusercontent.com/68471279/163261415-29ac1ea5-d5b7-453e-953d-41a3c2cfa1d6.png)

exactly the same dbo/fbx

one i can stand  on, one  I  can't with the only  difference is the texures.

![image](https://user-images.githubusercontent.com/68471279/163261937-4cd49d2c-57b9-4771-8f4d-4bb01acadf45.png)

**Comment by Kitakazi** [comment]
Hi I am also having issues with collision and importing custom models with the latest build. I figured I might as well post here since it could be related.

I have a very simple staircase model I made, I have imported and assigned Static and Polygon collision. When I drop the entity in my level the collision is fine. However if I rotate the entity at all the collision seems to become detached and not rotate correctly with the stairs. Causing the player to walk through the stairs and then a small distance away is the collision so then the player is walking up invisible stairs.

**Comment by ZakJudges** [comment]
Hi @casigus @Kitakazi Could you send me the dbo and fpe files that show the issue and I will take a look. My email address is zak@thegamecreators.com .Thanks :)

**Comment by casigus** [comment]
ohh i raged deleted them!

**Comment by Kitakazi** [comment]
sent

**Comment by ZakJudges** [comment]
@LeeBamberTGC The model that Kitakazi sent me shows the bug where the orientation of an objects collision shape doesn't match the orientation of the rendered object. Do you remember what the cause of that bug was?

**Comment by LeeBamberTGC** [comment]
@casigus Let me know if you are happy to close this, and you can create a new one if you find issues with the importers offset and rotation feature, thanks!

**Comment by Kitakazi** [comment]
@LeeBamberTGC @ZakJudges 
Hi Lee, the stair model is actually my model. I just replied to this thread as it was posted the same day I was having collision issues with my model and I didn't want to create duplicate issues. 

I can try and reimport it when I get home and see if its still botched. 

For more info, I most certainty did adjusted size and position/rotation in the importer. 
 Additionally, this model was originally an FBX it seems that when I saved it as GLTF the model gained the large size. I have issues when importing custom FBX models sometimes where the model imports but the model is completely corrupted, like the mesh is warped or if multiple meshes they will be in the wrong places or completely detached. However if I convert those said models from FBX to GLTF then they always imported fine.  This stair model was one of those that was corrupted on import as FBX. I converted to GLTF and imported, like I usually do with my custom assets, and that's when I noticed the collision bug. Never seen the bug before but I have this bug happen to other models since I posted that. Not sure if that is part of what is causing the collision to become detached or if that is an entirely separate issue.  

I do not have the original FBX of this model.
